src/components/SearchResultDetail/SearchResultDetail.tsx

```diff
@@ -32,6 +58,26 @@ const SearchResultDetail = ({ id }: { id: string }) => {
     findSearchResult();
   }, [id]);
 
+  const downloadPdf = async () => {
+    try {
+      setDownloadLoading(true);
+      const pdfString = await exportSearchResultPdfAction(searchResult.id);
+      if (!pdfString) {
+        return toast('Can not download the pdf version', 'error');
+      }
+
+      const fileName = `${searchResult.keyword}-${searchResult.searchEngine}-result.pdf`;
+      downloadBase64String({ base64String: pdfString, fileName, mimeType: mimeTypePDF });
+      setDownloadLoading(false);
+
+      toast(`File downloaded successfully as ${fileName}.`, 'success');
```

Comment:
MInor suggestion improvement:

```suggestion
      toast(`PDF export successfully created: ${fileName}.`, 'success');
```